src/components/ebay-pagination/examples/1-basic/template.marko

```diff
@@ -0,0 +1,13 @@
+<ebay-pagination aria-label-prev="Previous page" aria-label-next="Next page" curr-text="Results Pagination - Page 1">
+    <ebay-pagination-item href="#" previous></ebay-pagination-item>
+    <ebay-pagination-item href="#">1</ebay-pagination-item>
+    <ebay-pagination-item href="#" current>2</ebay-pagination-item>
+    <ebay-pagination-item href="#hh">3</ebay-pagination-item>
```

Comment:
If `hh` doesn't mean anything, remove it so we can keep this simple (here and in the other examples).

Reply:
Yeah I was just checking something and forgot to remove it.